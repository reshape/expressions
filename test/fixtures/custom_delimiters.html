x {% test %} x {% 1 + 1 %} x
<p class="x {% test %} x">x {% test %} x</p>
{{% 'x&x' %}}
