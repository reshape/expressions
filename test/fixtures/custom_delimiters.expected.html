x wow x 2 x
<p class="x wow x">x wow x</p>
x&x
